docs/pages/api/architecture.mdx

```diff
@@ -65,7 +65,11 @@ benefits, here's a quick breakdown:
    functionality, and customizability. Identity files can be generated through
    `tsh login`, `tctl auth sign`, or with
    [Machine ID](../machine-id/introduction.mdx).
- - Key Pair Credentials have a much simpler implementation than the first two
+ - DynamicIdentityFileCreds are IdentityFile credentials with support for
```

Comment:
I would use code style if we're directly referring to the name of a Go identifier. Otherwise, this seems like the underlying name of the abstraction (irrespective of language) is DynamicIdentityFileCreds.